doc/rules/avoid_preserve_whitespace_false.md

```diff
@@ -0,0 +1,40 @@
+# Double literal format
+
+## Rule id
+avoid_preserve_whitespace_false
+
+## Description
+Avoid use argument `preserveWhitespace: false` for `@Component` annotation in Angular, 
```

Comment:
`Avoid setting preserveWhitespace in Angular @Component annotations to false explicitly. Its default value is already false.`?

Reply:
fixed